recipe/meta.yaml

```diff
@@ -36,10 +36,25 @@ requirements:
     - traitlets
 
 test:
+  requires:
+    - nose
+    - setuptools
+    - wheel
+    - nose
+    - coverage
+    - nodejs
+    - phantomjs
```

Comment:
> Why would we not want to run these?

Mainly because these tests (and testing tools) are of insufficient quality, so most test failures are indicative only of problems in the tests themselves and not the validity of the code or package. They frequently require manual restart, which on appveyor for conda-forge can mean hours of delay. Running the tests in the packager should only test "was the package built correctly," and adding these flaky tests to conda-forge will not help that, but will cost a lot of time and effort.